pkgs/top-level/cubocore-packages.nix

```diff
@@ -0,0 +1,93 @@
+{ newScope, lxqt, lib, libsForQt5 }:
```

Comment:
Please format using 2 spaces.

Reply:
Most of the lines below are not formatted with the right amount of spaces. You can use nixpkgs-fmt to get it right.